app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -126,6 +126,14 @@ class Config {
     return ServiceAccountCredentials.fromJson(json.decode(rawValue));
   }
 
+  /// The names of autoroller accounts for the repositories.
+  ///
+  /// These accounts should not need reviews before merging.
```

Comment:
Can we add a see also link to https://github.com/flutter/flutter/wiki/Autorollers?

Reply:
Good call, done.